test/unit/text-field/index.test.js

```diff
@@ -80,7 +80,7 @@ test('#componentDidUpdate calls setValue if state.value updates', () => {
 test('#componentDidUpdate does not call setValue if another property updates', () => {
   const wrapper = shallow(<TextField label='my label'><Input /></TextField>);
   wrapper.instance().foundation_.setValue = td.func();
-  wrapper.setState({dir: 'rtl'});
+  wrapper.setState({dir: 'rtl'}); // could be anything
```

Comment:
What does "could be anything" mean?

Reply:
I'll rewrite as  // could be any property/value pair. Or should i just remove the comment altogether?